admin_guide/high_availability.adoc

```diff
@@ -207,40 +229,38 @@ The list of command options described here are a subset that are relevant to thi
             <number> = a number ([0-9]*).
 ////
 
-The `oadm ipfailover` command ensures that a failover pod runs on each of
-the nodes matching the constraints or label used. This pod uses VRRP (Virtual
-Router Redundancy Protocol) with link:http://www.keepalived.org/[Keepalived] to ensure that the service on the
-watched port is available, and, if needed, Keepalived will automatically float
-the VIPs if the service is not available.
-
-[[configuring-a-highly-available-routing-service]]
-=== Configuring a Highly-available Routing Service
-The following steps describe how to set up a highly-available router environment
-with IP failover:
-
-. Label the nodes for the service. This step can be optional if you run the
-service on any of the nodes in your Kubernetes cluster and use VIPs that can
-float within those nodes. This process may already exist within a complex
-cluster, in that nodes may be filtered by any constraints or requirements
-specified (e.g., nodes with SSD drives, or higher CPU, memory, or disk
-requirements, etc.).
-+
-The following example defines a label as router instances that are servicing
-traffic in the US west geography *ha-router=geo-us-west*:
-+
+[[virtual-ips]]
+=== Virtual IP Addresses
+*Keepalived* manages a set of virtual IP addresses. The admin must make sure that all these addresses:
+
+* Are accessible on the configured hosts from outside the cluster.
+* Are not used for any other purpose within the cluster.
+
+
+*Keepalived* on each node determines whether the needed service is running. If it is, VIPs are supported and *Keepalived* participates in the negotiation to determine which node will serve the VIP. For a node to participate, the service must be listening on the watch port on a VIP or the check must be disabled.
+
+[NOTE]
 ====
-----
-$ oc label nodes openshift-node-{5,6,7,8,9} "ha-router=geo-us-west"
-----
+Each VIP in the set may end up being served by a different node.
 ====
 
-. {product-title}'s *ipfailover* internally uses *keepalived*, so ensure that
-multicast is enabled on the nodes labeled above and that the nodes can accept
-network traffic for 224.0.0.18 (the VRRP multicast IP address). Depending on
-your environment's multicast configuration, you may need to add an `iptables`
-rule to each of the above labeled nodes. If you do need to add the `iptables`
-rules, please also ensure that the rules persist after a system restart:
-+
+[[kepalived-multicast]]
+=== Keepalived Multicast
+{product-title}'s ipfailover internally uses *keepalived*. Please ensure that multicast is enabled on the nodes labeled above and that the nodes can accept network traffic for 224.0.0.18 (the VRRP multicast IP address).
```

Comment:
The "Please ensure" sentence would probably server better in an "Important" box.